devices/pine64-pinephone/README.adoc

```diff
@@ -50,8 +46,6 @@ Its colour or pattern can be used to infer the status.
 
 During boot:
 
- * Red as early as possible in our U-Boot build
- * Yellow when our U-Boot "boot script" ran
  * Green by the kernel whenever it sets it up
```

Comment:
```suggestion
 * Green by the kernel whenever it is launched by the platform firmware
```

Reply:
The current phrasing is right, even if it feels awkward.

The LED is set green by the kernel when the module initialization happens for the LEDs. This means that between the moment it's launched by the platform firmware, and [built-in] modules initialization, there is a span of time where the kernel is already running, but the LED is in whichever state the firmware left it.

Reply:
Ok gotcha, I was confused about what both of the "it"s meant.

Better phrasing might be "Green when the kennel initializes the LED".